gapic-generator/lib/gapic/presenters/method_presenter.rb

```diff
@@ -160,6 +160,15 @@ def server_streaming?
 
       def paged?
         return false if server_streaming? # Cannot page a streaming response
+
+        # HACK: Two specific RPCs should not be paged.
+        # This is an intentionally hard-coded exception (and a temporary one,
+        # to be removed when these methods no longer conform to AIP-4233.) For
```

Comment:
add date and who wrote this, one thing that always comes up when reading HACKs and TODOs is who and when. Having to go look it up in blame sucks.

Reply:
👍